### PR TITLE
Cookbook

### PR DESCRIPTION
**Pull request by other019**

### CODE REVIEW

File: cookbook/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# This is cookbook
+
+The cookbook is a place where we put recipes for things around MrChecker framework.
+
+* [small project template](./small_project_template) - set of basic CI/CD tools to conduct a small project
+* [huge project template](./huge_project_template) - set of best practices with examples for conducting huge projects
```
Comment (Pyskaczu) [inline cookbook/readme.md:6]: Can we add model-based-testing as well? Or does model-based-testing need working on?

**Review by Pyskaczu — APPROVED** [review]
If possible add model-based-testing to cookbook/readme.md and merge.

File: cookbook/readme.md (hunk 1 above)
Comment (other019) [reply cookbook/readme.md:6]: I believe model base testing is already merged to Develop so I'm pretty suprised the merge conflict is not raised.

Comment (other019) [reply cookbook/readme.md:6]: NVM model based testing does not have this table of content page. Thats why there is no conflict

**Pull request merged**